tests/core/pyspec/eth2spec/debug/random_value.py

```diff
@@ -102,19 +103,23 @@ def get_random_ssz_object(
             get_random_ssz_object(rng, elem_type, max_bytes_length, max_list_length, mode, chaos)
             for _ in range(typ.vector_length())
         )
-    elif issubclass(typ, List) or issubclass(typ, Bitlist):
-        length = rng.randint(0, min(typ.limit(), max_list_length))
+    elif issubclass(typ, List) or issubclass(typ, ProgressiveList) or issubclass(typ, Bitlist):
+        limit = max_list_length
+        if (
+            not issubclass(typ, ProgressiveList) and typ.limit() < limit
+        ):  # SSZ imposes a hard limit on lists, we can't put in more than that
+            limit = typ.limit()
```

Comment:
Let's put the comment on the line above (and re-run `make lint`) so this isn't so ugly.